pkg/cloud/services/interfaces.go

```diff
@@ -37,3 +37,9 @@ type EC2MachineInterface interface {
 	TerminateInstanceAndWait(instanceID string) error
 	DetachSecurityGroupsFromNetworkInterface(groups []string, interfaceID string) error
 }
+
+// SecretsManagerMachineInterface encapsulated the methods exposed to the
+// machine actuator
```

Comment:
"SecretsManagerMachineInterface contains methods for working with encrypted bootstrap data." or something like that.